### DISCUSSION TITLE
how send image

### DISCUSSION THREAD

**Discussion by mehradamiri** [pocketbase/pocketbase#3206] [Q&A]
I'm using sveltekit, 
in my +page.server.js im trying to send an image but it doesn't work and the cover field in data base will be set to N/A
```js
	
const data = {
	name: body.name,
	writer: locals.user?.id,
	main_text: body.main_text,
	discription: body.discription,
	cover: body.cover
		};
console.log(data.cover);
 /* File {
  size: 4509,
  type: 'image/png',
  name: 'pher.png',
  lastModified: 1693151179464
}
*/
			const newBlog = await locals.pb.collection('blogs').create(data);
			
```

**Comment by ganigeorgiev** [comment]
What version of the JS SDK do you use?
Where your code is executed - in the browser or server-side?

Note that support for submitting File/Blob as plain object prop value was added in [JS SDK v0.17+](https://github.com/pocketbase/js-sdk/releases/tag/v0.17.0).

For earlier versions you have to prepare a [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData) instance as body.
